### PR TITLE
Login integration

### PR DESCRIPTION
**Pull request by maheshp2002**

### CODE REVIEW

File: client/idp/src/app/service/account.service.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { Injectable } from '@angular/core';
 })
 export class AccountsService {
 
-    url = "https://localhost:7036/api/accounts";
+    url = "https://localhost:7264/api/user/Accounts";
```
Comment (mishelshaji) [inline client/idp/src/app/service/account.service.ts]: Change the URL to lowercase.

File: client/idp/src/app/service/account.service.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { Injectable } from '@angular/core';
 })
 export class AccountsService {
```
Comment (mishelshaji) [inline client/idp/src/app/service/account.service.ts:7]: Format the code.

File: client/idp/src/app/user/signin/signin.component.ts [hunk 3]
```diff
@@ -1,13 +1,32 @@
...
+    private service: AccountsService,
+    private router: Router,
+    private tokenHelper: TokenHelper) { }
+
+  onSubmit() {
+    this.service.login(this.model).subscribe({
+      next: (response: any) => {
+        this.tokenHelper.setToken(response.result);
+        this.router.navigateByUrl('/user/profile');
+        console.log(response.result);
```
Comment (mishelshaji) [inline client/idp/src/app/user/signin/signin.component.ts]: Remove blank lines and console.log statements.

File: client/idp/src/app/app.module.ts [hunk 4]
```diff
@@ -26,9 +29,13 @@ import { NavbarManagerComponent } from './shared/navbar-manager/navbar-manager.c
     FooterComponent,
     FontAwesomeModule,
     NgCircleProgressModule.forRoot(),
-    NgChartsModule
+    NgChartsModule,
+    HttpClientModule,
```
Comment (mishelshaji) [inline client/idp/src/app/app.module.ts]: Remove the trailing comma.

File: server/Idp/IDP.Domain/Models/Idp.cs [hunk 5]
```diff
@@ -13,10 +13,6 @@ public class Idp
         [StringLength(200)]
         public string? Name { get; set; }
```
Comment (mishelshaji) [inline server/Idp/IDP.Domain/Models/Idp.cs:14]: The name is already there in the ApplicationUser table. Why did you add it again here?

File: server/Idp/IDP.Domain/Models/Manager.cs [hunk 6]
```diff
@@ -1,11 +1,14 @@
...
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace Idp.Domain.Models
 {
+    [Index(nameof(EmployeeId), IsUnique = true)]
+
     public class Manager
```
Comment (mishelshaji) [inline server/Idp/IDP.Domain/Models/Manager.cs:12]: The name is already there in the ApplicationUser table. Why did you add it again here?

File: server/Idp/Idp.Service/Dto/ProfileDto.cs [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+        public string LastName { get; set; }
+
+        public string Email { get; set; }
+
+        public string Department { get; set; }
+
+        public string Designation { get; set; }
+
+        public int EmployeeId { get; set; }
+
```
Comment (mishelshaji) [inline server/Idp/Idp.Service/Dto/ProfileDto.cs:24]: Remove the extra blank line.

File: server/Idp/Idp.WebApp/Areas/Admin/Controllers/ManagerController.cs [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+            return Ok(result);
+        }
+
+        [HttpPost]
+        [ProducesResponseType(typeof(ViewManagerDto[]), StatusCodes.Status201Created)]
+        public async Task<ActionResult> Post(CreateManagerDto dto)
+        {
+            var result = await _service.CreateAsync(dto);
+            return Ok(result);
+            //return CreatedAtAction(nameof(GetAsync), new { id = result.Id }, result);
```
Comment (mishelshaji) [inline server/Idp/Idp.WebApp/Areas/Admin/Controllers/ManagerController.cs]: Remove the comment.

File: server/Idp/Idp.WebApp/Areas/User/Controllers/AccountController.cs [hunk 9]
```diff
@@ -37,5 +40,14 @@ public async Task<IActionResult> PostLogin(LoginDto dto)
 
             return BadRequest(result.Errors);
         }
+
+        [Authorize(Roles = "User")]
+        [HttpGet("profile")]
+        public async Task<IActionResult> GetProfile()
```
Comment (mishelshaji) [inline server/Idp/Idp.WebApp/Areas/User/Controllers/AccountController.cs:47]: Add response types.

**Review by mishelshaji — CHANGES_REQUESTED** [review]

File: server/Idp/IDP.Domain/Models/Manager.cs (hunk 6 above)
Comment (maheshp2002) [reply server/Idp/IDP.Domain/Models/Manager.cs:12]: This one is not  login, this is different table, when removing name, it shows error. Can we use Application user for other tables than user

**Review by mishelshaji — APPROVED** [review]
